tests/unit/test_config.py

```diff
@@ -1081,18 +1080,23 @@ def test_apply_cloud_providers_config_extend(self):
                          [{'extends': 'my-dev-envs:ec2',
                            'location': 'us-east-1',
                            'user': 'ec2-user@mycorp.com'
-                          }],
+                           }],
                      'my-dev-envs':
                          [{'id': 'ABCDEFGHIJKLMNOP',
                            'user': 'user@mycorp.com',
                            'location': 'ap-southeast-1',
                            'key': 'supersecretkeysupersecretkey',
                            'driver': 'ec2'
-                          },
+                           },
                           {'apikey': 'abcdefghijklmnopqrstuvwxyz',
                            'password': 'supersecret',
                            'driver': 'linode'
-                          }],
+                           },
+                          {'id': 'a-tencentcloud-id',
+                           'key': 'a-tencentcloud-key',
+                           'location': 'ap-guangzhou',
+                           'driver': 'tencentcloud'
+                           }],
```

Comment:
Did you mean  to leave this, or do you want to make it
```
  },
]
```
(TBH, I'd probably just reformat this whole my-dev-envs to something like...)

```
'my-dev-envs': [
    {'id': 'ABCD...',
     'user': ...
     },
    {'apikey': ...
     },
    {'id': '...',
     },
]
```